### DISCUSSION TITLE
Unable to import stylesheet of library

### DISCUSSION THREAD

**Discussion by rishavbharti** [postcss/postcss#1934] [Q&A] [answered]
Hi there,

I'm in the process of creating a React UI library. I'm utilizing PostCSS for transforming CSS.

I'm encountering an issue with importing the stylesheet. Could someone please assist me in troubleshooting this?

[Link to CodeSandbox](https://codesandbox.io/p/github/rishavbharti/poc/main)

**Comment by ai** [comment]
Sorry, the project is too big

**Reply by ai** [reply]
It is still too big.

Post here an error message and relevant source codes.

**Reply by ai** [reply]
This error is not part of PostCSS, but the part of your bundler.

I can only guest, but seems like you need to add `styles.css` to `package.json`→`exports` (see the right syntax in Google).

**Reply by rishavbharti** [reply, marked as answer]
Yes, found a way https://github.com/vitejs/vite/discussions/2657. 
Thanks and sorry for the trouble
